StoryLine.Selenium.Runner/SimpleAdviceInvestment/Actions/GoToLoginPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using StoryLine.Contracts;
+using StoryLine.Selenium.Actions;
+using StoryLine.Selenium.Runner.SimpleAdviceInvestment.Pages;
+using StoryLine.Utils.Expectations;
+using System;
+
+namespace StoryLine.Selenium.Runner.SimpleAdviceInvestment.Actions
+{
+    class GoToLoginPage : IActionBuilder, IAction
```

Comment:
@SLebedzeu i would suggest to put `public` keyword here.

I think it would be logical include login actions too. Navigation in majority of cases is followed by submitting valid credentials. So it would rather implement action `LoginToPfp` (e.g. combine with action below).